### PR TITLE
feat: add query local min-gas-price

### LINKED ISSUE #2629: add query node config: `minimum-gas-prices` in app.toml
# Problem
Node's local `min-gas-price` in `app.toml` should be queryable. 

Register NodeService, so that `/cosmos/base/node/v1beta1/config` gRPC query can expose node operator's configured minimum-gas-price.

ref: [proto query config defination in sdk](https://github.com/cosmos/cosmos-sdk/blob/v0.45.16-ics/proto/cosmos/base/node/v1beta1/query.proto).

# Closing criteria
Users can query a node's  minimum-gas-price through `/cosmos/base/node/v1beta1/config` .

# Problem details

### PR DESCRIPTION
**Pull request by yaruwangway**
## Description

Closes: #2629

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [x] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if API, client, or state breaking change (i.e., requires minor or major version bump)
* [x] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] Provided a link to the relevant issue or specification
* [ ] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [x] Included the necessary unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
* [x] Added a changelog entry in `.changelog` (for details, see [contributing guidelines](../../CONTRIBUTING.md#changelog))
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [x] Reviewed "Files changed" and left comments if necessary 
* [x] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed all author checklist items have been addressed 
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: tests/e2e/e2e_rest_regression_test.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ const (
 	slashingParamsModuleQueryPath       = "/slashing/parameters"
 	stakingParamsModuleQueryPath        = "/staking/parameters"
 	missingPath                         = "/missing_endpoint"
+	localMinGasPriceQueryPath           = "cosmos/base/node/v1beta1/config"
```
Comment (mpoke) [inline tests/e2e/e2e_rest_regression_test.go]: This is never used.

**Review by mpoke — CHANGES_REQUESTED** [review]

File: tests/e2e/e2e_rest_regression_test.go (hunk 1 above)
Comment (yaruwangway) [reply tests/e2e/e2e_rest_regression_test.go]: I dont understand, what do you mean ?

Comment (mpoke) [reply tests/e2e/e2e_rest_regression_test.go]: IIUC, this constant is never used in the test.

Comment (yaruwangway) [reply tests/e2e/e2e_rest_regression_test.go]: you are right, my bad, added test.

**Comment by yaruwangway** [comment]
@mpoke if this PR backport this to v10 ?

**Review by mpoke — APPROVED** [review]

**Review by mmulji-ic — APPROVED** [review]

**Review by bitcoinfinancier — APPROVED** [review]

**Pull request merged**